include/wx/ribbon/bar.h

```diff
@@ -68,7 +68,7 @@ class WXDLLIMPEXP_RIBBON wxRibbonBarEvent : public wxNotifyEvent
     void SetPage(wxRibbonPage* page) {m_page = page;}
 
 protected:
-    wxRibbonPage* m_page;
+    wxRibbonPage* m_page{ nullptr };
```

Comment:
We probably need to standardize on whether we use `{initializer}` or `= initializer`.

There is no difference between them for primitive types, including pointers, but the former does have the advantage of working better for classes. Subjectively I find the latter more readable, however.

Any thoughts/opinions/preferences?

Reply:
I prefer `{}` myself because it's more "modern" looking, but `=` is totally OK.  I happy to use either.

With `{}`, I've gotten into the habit of using spaces like `{ nullptr }` to make it easier to read.  Just my style, I'm happy to use a different style.

Reply:
I find `=` more readable, e.g. I agree that

```c++
T* m_ptr{nullptr};
```

is not very readable without spaces, but 

```c++
T* m_ptr = nullptr;
```

is IMO just fine on its own.

And I think in modern C++, i.e. with guaranteed copy elision, there are few drawbacks to using it compared to `{}`. The latter does forbid narrowing conversions, but wx code is supposed to be compiled with warning options catching the most egregious of them and you're unlikely to have really unwanted conversions when writing simple member initializers IMHO.

So if there are no big objections to it, I'd rather use `=`. TIA!

Reply:
An OT and outsider's comment: the new initializer `{}` used without any contents will emulate the default "0", "false", "nullptr" initialization for primitive types, and calls the default ctor for objects (which is by default since C++98).  So basically for most type `T`, `T obj{};` should do the reasonable default initialization.  This also matches the behavior in newer languages with default initialization like Go.  Though admittedly this is not a widely applied convention, so just my 2 cents.

Reply:
Yes, this is correct but, again, `{}` loses to `= 0` from readability point of view IMO.

I also don't think we're going to switch to writing `int i{};` instead of `int i = 0;` for local variables and using the same syntax for them and member variables is also a (minor) advantage.

Reply:
Indeed, and I suspect that it's exactly because of this difference between old "=" code that prevents its wider use, unless this was done from the beginning.  Totally makes sense for existing codebase not to switch to that.

Reply:
Got it, will use `=`.